README.md

```diff
@@ -120,21 +120,21 @@ In case you want to import this library directly (without having to use npm or y
 
 ```html
 <script type="module">
-  import { initialize, Event } from 'https://unpkg.com/@harnessio/ff-javascript-client-sdk@1.4.4/dist/sdk.client.js'
+  import { initialize, Event } from 'https://unpkg.com/@harnessio/ff-javascript-client-sdk@1.4.10/dist/sdk.client.js'
```

Comment:
No need to change, just thinking outloud... We should look at what we're publishing to npm; there's an argument to be made about only publishing the contents of the dist folder. Also we should have at least an index file so we can simplify this to `import { initialize, Event } from 'https://unpkg.com/@harnessio/ff-javascript-client-sdk`. We then wouldn't need to change this between releases.

Reply:
make sense